models.py

```diff
@@ -1,22 +1,12 @@
-from keras.models import *
-from keras.layers import *
-from keras import layers
-from keras.regularizers import l2
+from tensorflow.keras.models import Model
+from tensorflow.keras.layers import Conv2D, Concatenate, ZeroPadding2D, BatchNormalization, Activation, MaxPooling2D, UpSampling2D, Input, Layer
+from tensorflow.keras import layers
+from tensorflow.keras.regularizers import l2
+import tensorflow as tf
 
 resnet50_Weights_path='./pretrained_model/resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5'
-IMAGE_ORDERING ='channels_last'
```

Comment:
This should be passed as a variable, and ideally use tensorflow's default.

TODO: default to `image_data_format` from keras' config (`~/.keras/keras.json`)